packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -358,32 +358,40 @@ export abstract class BasePHP implements IsomorphicLocalPHP, Disposable {
 	}
 
 	#initWebRuntime() {
+		this.writeFile(
+			'/internal/auto_prepend_file.php',
+			`<?php
+			foreach (glob('/internal/preload/*.php') as $file) {
+				require_once $file;
+			}
+		`
+		);
+		this.setPhpIniEntry(
+			'auto_prepend_file',
+			'/internal/auto_prepend_file.php'
+		);
```

Comment:
Nice, from the original discussion I assumed that we would use WP hooks, but `auto_prepend_file` seems like a great solution.